version.md

```diff
@@ -0,0 +1,78 @@
+# Versioning endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents a single version, matching a single item.
+
+Whether a version is accessible depends on `versioning.item.history.view.admin` configuration.  
+If this is set to true, the version can only be retrieved if the user is an admin of the related item
+
+## Create version
+
+TODO
+
+## Get single version
+
+**GET /api/versioning/version/<:versionId>**
```

Comment:
this should be `/versioning/versions` or `/versioning/itemversions`

Reply:
I agree and overlooked this. It should be a plural, not a singular

Reply:
Good catch, I've adjusted this